packages/server/src/sdk/app/tables/external/utils.ts

```diff
@@ -22,12 +22,16 @@ export function cleanupRelationships(
   tables: Record<string, Table>,
   oldTable?: Table
 ) {
+  if (!oldTable) {
+    return
+  }
```

Comment:
Would prefer to see `oldTable` become a non-optional parameter and push the handling of that scenario up to the callers. As it is now it's easy to call the function assuming `oldTable` is fine to omit, only to realise the function then does nothing.